misc.php

```diff
@@ -93,6 +98,17 @@ function vipgoci_sysexit(
 		$irc
 	);
 
+	/*
+	 * If running SysExitTest.php unit-test, return
+	 * with exit status.
+	 */
+	if (
+		( function_exists( 'vipgoci_unittests_check_indication_for_test_id' )) &&
```

Comment:
Ideally, it should not be needed to modify the original function to make the test pass/work. The tests should always test against the actual application behavior. However, in this context, with the ``exit`` usage, I understand it's not possible for the moment.